changelog/8750.improvement.md

```diff
@@ -0,0 +1 @@
+Changed ``DietClassifier`` class so that we are able to persist and load sparse feature sizes. This will be helpful for incremental training.
```

Comment:
is this actually just an internal change? In that case it should be `misc` instead of `improvement`. 
```suggestion
Changed `DietClassifier` class so that we are able to persist and load sparse feature sizes. This will be helpful for incremental training.
```

Reply:
Great 👍  agree it's an internal change and it should be ``misc``.